### ISSUE TITLE
[BUG]: Unable to add ScenePalace

### ISSUE THREAD

**Issue by MaleNurse** [Prowlarr/Prowlarr#369]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When adding ScenePalace.info the following error occurs:

`Unable to connect to indexer. We are being redirected to the login page. Most likely your session expired or was killed. Try testing the indexer in the settings.`

### Expected Behavior

The ability to add ScenePalace as an indexer. It is still working in Jackett, and I'm able to log in through the webpage (albeit a page w/ a weird captcha)

### Environment

```markdown
- OS: Ubuntu 20.04.2 LTS
- Prowlarr: 0.1.0.608
- Docker Install: Yes, 20.10.7 running LinuxServer.io's image
- Using Reverse Proxy: Yes, but tested both
- Browser: Vivaldi 4.0.2312.41
```

### What branch are you running?

Develop

### Anything else?

[Relevant Trace Log](https://hastebin.com/raw/vofetiquja)

[AB#1291](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1291)

**Comment by paul-chambers** [comment]
Unfortunately, I don't have any invites to give at the moment. But I did contact the staff at ScenePalace to see if they would provide one to the dev team.

**Comment by MaleNurse** [comment]
Ditto here, about 10d ago but haven't heard anything back unfortunately

**Comment by Qstick** [comment]
Thanks @paul-chambers and @MaleNurse. Let us know if you hear anyting.

**Comment by stedj34** [comment]
QStick, I have an invite--how should I send it? Same issue occurs with p2pElite (very similar site). Would you need an invite for there as well?

**Comment by stedj34** [comment]
> QStick, I have an invite--how should I send it? Same issue occurs with p2pElite (very similar site). Would you need an invite for there as well?

@Qstick I just realized I didn't actually tag you

**Comment by Qstick** [comment]
qstick @ protonmail.com

**Comment by stedj34** [comment]
@Qstick just sent you invites to ScenePalace and p2pElite

**Comment by Qstick** [comment]
Should be fixed with c53e0054ee
